resources/definitions/eazao_zero.def.json

```
@@ -0,0 +1,105 @@
+        },
+        "machine_end_gcode":
+        {
+            "default_value": "M107 ; Fan off\nG91 ;relative positioning\nG1 E-10 F300 ;retract the filament a bit before lifting the nozzle, to release some of the pressure\nG1 Z+3 E-5  F{speed_travel} ;move Z up a bit and retract filament even more\nG28 X0 Y0;move X Y to min endstops\nG90 ;Absolute positioning\nM84 ;steppers off\n"
+        },
+        "machine_max_feedrate_x": { "value": 100 },
+        "machine_max_feedrate_y": { "value": 100 },
+        "machine_max_feedrate_z": { "value": 5 },
+        "machine_max_feedrate_e": { "value": 25 },
+
+        "machine_max_acceleration_x": { "value": 500 },
+        "machine_max_acceleration_y": { "value": 500 },
+        "machine_max_acceleration_z": { "value": 50 },
+        "machine_max_acceleration_e": { "value": 500 },
+        "machine_acceleration": { "value": 300 },
+        "acceleration_print": { "value": 300 },
+        "acceleration_travel": { "value": 300 },
+        "acceleration_enabled": { "value": false },
+
+        "machine_max_jerk_xy": { "value": 10 },
+        "machine_max_jerk_z": { "value": 0.3 },
+        "machine_max_jerk_e": { "value": 5 },
+        "jerk_print": { "value": 10 },
+        "jerk_travel": { "value": "jerk_print" },
+        "jerk_travel_layer_0": { "value": "jerk_travel" },
+        "jerk_enabled": { "value": false },
+
+        "speed_print": { "value": 20.0 },
+        "speed_z_hop": { "value": "machine_max_feedrate_z" },
+
+        "optimize_wall_printing_order": { "value": "True" },
+        "material_initial_print_temperature": { "value": "0" },
+        "material_final_print_temperature": { "value": "0" },
+
+        "retraction_hop_enabled": { "value": "false" },
+        "retraction_hop": { "value": 0.2 },
+        "retraction_combing": { "default_value": "noskin" },
+        "retraction_combing_max_distance": { "value": 0 },
+
+        "travel_avoid_other_parts": { "value": true },
+        "travel_avoid_supports": { "value": false },
+        "travel_retract_before_outer_wall": { "value": false },
+
+        "retraction_enable": { "value": false },
+        "retraction_speed": { "value": 25 },
+        "retraction_amount": { "value": 7 },
+        "retraction_count_max": { "value": 100 },
+        "retraction_extrusion_window": { "value": 10 },
+        "retraction_min_travel": { "value": 1.5 },
```

```suggestion
        "retraction_min_travel": { "value": 1.5 }
```
It's invalid json with the trailing , 

Did you test this if it actually worked in Cura? I would also have failed there.

---

I have tested these parameters in cura before, and now I will check them again and give the correct reply later.